packages/SketchMorph2-Core.package/M2Backend.class/instance/drawRectangle.st

```diff
@@ -1,6 +1,10 @@
 drawing
 drawRectangle
 	self isErasing: false.
-	self pen sourceForm: ((Form extent: self rectangleBrushSize) fillBlack).
+	self pen sourceForm:
+		(Form
+			extent: self rectangleBrushSize
+			offset: (self rectangleBrushSize / 2) floor negated)
```

Comment:
I'd suggest extracting this calculation to its own method, taking a point `aPoint` and returning `(aPoint / 2) floor negated`. (Especially since this calculation is used twice.)